src/solution.c

```diff
@@ -396,6 +401,53 @@ static void sol_thd_sleep(systime_t *deadline, systime_t interval)
   chSysUnlock();
 }
 
+/** Calculates Iono and Tropo correction
+ * \param n_ready_tdcp The number of measurements written to `m_tdcp`
+ * \param nav_meas_tdcp Pointer to measurements array */
+void calc_iono_tropo(u8 n_ready_tdcp,
```

Comment:
@benjamin0 In our navigation_message structures we very clearly separate pseudrorange / raw_pseudorange and carrier_phase / raw_carrier_phase.

Anything to do with RTK should be using raw_* values, only SPP uses the non-raw values.